### PR TITLE
Add cache config strategies.yaml generation

### PR DESCRIPTION
**Pull request by rob05c**
Adds t3c strategies.yaml config generation. The file is always created (as is parent.config), but by default it isn't used. A new flag is added to `t3c-apply`, `--use-strategies` to specify using the parent_select plugin or core Strategies. This flag should be considered experimental.

- [x] This PR fixes #5804

## Which Traffic Control components are affected by this PR?
- Traffic Ops ORT

## What is the best way to verify this PR?
Run unit tests. Run t3c, verify all config files are generated as-expected without strategies. Pass `--use-strategies`, verify remap.config has strategy directives. Verify generated strategies file is correct.

## If this is a bug fix, what versions of Traffic Control are affected?
Not a bug fix.

## The following criteria are ALL met by this PR
- [x] This PR includes tests OR I have explained why tests are unnecessary
- [x] This PR includes documentation OR I have explained why documentation is unnecessary
- [x] This PR includes an update to CHANGELOG.md OR such an update is not necessary
- [x] This PR includes any and all required license headers
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://www.apache.org/security/) for details)

## Additional Information

### CODE REVIEW

**Review by jpappa200 — CHANGES_REQUESTED** [review]
1 Unused import otherwise looks good to me.

File: lib/go-atscfg/remapdotconfig.go [hunk 1]
```diff
@@ -21,7 +21,9 @@ package atscfg
 
 import (
 	"errors"
+	"fmt"
```
[the remark this answers is not in the document]
Comment (rob05c) [reply lib/go-atscfg/remapdotconfig.go]: Fixed

**Review by jpappa200 — APPROVED** [review]
👍

**Comment by rawlinp** [comment]
@rob05c there are conflicts that need resolved

**Pull request merged**